tsconfig.json

```diff
@@ -0,0 +1,66 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "esnext",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "module": "es2015",                       /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "lib": ["dom", "es2015", "es2016"],       /* Specify library files to be included in the compilation. */
+    "jsx": "react-native",                    /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+    "declaration": true,                      /* Generates corresponding '.d.ts' file. */
+    "sourceMap": false,                       /* Generates corresponding '.map' file. */
+    "rootDir": "src",                         /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    "outDir": "dist",                         /* Redirect output structure to the directory. */
+    "importHelpers": true,                    /* Import emit helpers from 'tslib'. */
+    // "allowJs": true,                       /* Allow javascript files to be compiled. */
```

Comment:
maybe better to remove commented out code?

Reply:
done! :)